src/backend/commands/explain.c

```diff
@@ -71,8 +71,8 @@ static void report_triggers(ResultRelInfo *rInfo, bool show_relname,
 
 #ifdef USE_ORCA
 static void ExplainDXL(Query *query, ExplainState *es,
-							const char *queryString,
-							ParamListInfo params);
+		   const char *queryString,
+		   ParamListInfo params);
```

Comment:
Remove changes that not connected with your patch. Apply pgindent changes only for modified lines.